### PR TITLE
ci: rename master to main

### PR DESCRIPTION
**Pull request by travisgroth**
## Summary

Update CI for renaming master branch to main.

## Related issues

n/a

## Checklist

- [ ] reference any related issues
- [ ] updated docs
- [ ] updated unit tests
- [ ] updated UPGRADING.md
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

**Review by desimone — CHANGES_REQUESTED** [review]
I believe there are still many several references in the integration tests (though the duplicate tag should avoid breakage I think?)  and docs. Maybe best to do in one sweep?

```
examples/kubernetes/pomerium-values.yaml
16:    connectionString: rediss://pomerium-redis-master.pomerium.svc.cluster.local

examples/kubernetes/pomerium-authenticate.yml
30:        - image: pomerium/pomerium:master

examples/kubernetes/pomerium-proxy.yml
32:        - image: pomerium/pomerium:master

examples/kubernetes/pomerium-databroker.yml
31:        - image: pomerium/pomerium:master

examples/kubernetes/pomerium-authorize.yml
30:        - image: pomerium/pomerium:master

examples/kubernetes/pomerium-certificates.yaml
34:    - pomerium-redis-master.pomerium.svc.cluster.local

scripts/protoc
33:download $_protoc_3pp_path googleapis https://github.com/googleapis/googleapis/tarball/master

examples/helm/helm_gke.sh
3:# curl https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get | bash

README.md
7:[![LICENSE](https://img.shields.io/github/license/pomerium/pomerium.svg)](https://github.com/pomerium/pomerium/blob/master/LICENSE)

docs/reference/readme.md
1091:- [pkg/grpc/databroker/databroker.proto](https://github.com/pomerium/pomerium/blob/master/pkg/grpc/databroker/databroker.proto)
1095:- [pkg/databroker/memory](https://github.com/pomerium/pomerium/tree/master/pkg/databroker/memory)
1131:- sentinel: `redis+sentinel://[:password@]host:port[,host2:port2,...]/[master_name[/db]][?param1=value1[&param2=value2&...]]`
1672:For more details, see our [mTLS example repository](https://github.com/pomerium/pomerium/tree/master/examples/mutual-tls) and the [Upstream mTLS With Pomerium](/guides/upstream-mtls.md) guide.
1805:For example, assuming you have [generated an ES256 key](https://github.com/pomerium/pomerium/blob/master/scripts/generate_self_signed_signing_key.sh) as follows.
1847:[script]: https://github.com/pomerium/pomerium/blob/master/scripts/generate_wildcard_cert.sh

docs/docs/topics/getting-users-identity.md
123:[docker-compose.yml]: https://github.com/pomerium/pomerium/blob/master/docker-compose.yml

docs/guides/cloud-run.md
14:This recipe's sources can be found [on github](https://github.com/pomerium/pomerium/tree/master/examples/cloudrun)
41:[Dockerfile](https://github.com/pomerium/pomerium/blob/master/.github/Dockerfile-cloudrun) Based on [vals-entrypoint](https://github.com/pomerium/vals-entrypoint)

docs/guides/istio.md
22:- This configuration uses the Pomerium Ingress Controller for [north-south traffic]. This guide uses our [Helm chart](https://github.com/pomerium/pomerium-helm/tree/master/charts/pomerium) as detailed in [Install Pomerium using Helm]. We'll cover the values needed to configure the controller with an Istio service mesh, but you can refer to the [documentation][Pomerium Ingress Controller] for a complete overview of the controller spec.

docs/guides/nginx.md
14:This recipe's sources can be found [on github](https://github.com/pomerium/pomerium/tree/master/examples/nginx)
39:- A copy of the [example repo](https://github.com/pomerium/pomerium/tree/master/examples/nginx) checked out

docs/guides/tcp.md
15:The source files from this guide can be found on [GitHub](https://github.com/pomerium/pomerium/tree/master/examples/tcp/).
40:* A copy of the [example repo](https://github.com/pomerium/pomerium/tree/master/examples/tcp/) checked out

docs/guides/traefik-ingress.md
13:This guide's sources can be found [on github](https://github.com/pomerium/pomerium/tree/master/examples/traefik-ingress).
40:- A copy of the [example repo](https://github.com/pomerium/pomerium/tree/master/examples/traefik-ingress) checked out

docs/docs/k8s/ingress.md
22:- Pomerium [Helm Chart](https://github.com/pomerium/pomerium-helm/tree/master/charts/pomerium) v25.0.0+
375:- [Pomerium Helm Chart README: Pomerium Ingress Controller](https://github.com/travisgroth/pomerium-helm/tree/master/charts/pomerium#pomerium-ingress-controller)


docs/docs/tcp/rdp.md
54:    wget https://github.com/pomerium/pomerium/blob/master/examples/tcp/pomerium-tunnel.sh

docs/docs/releases.md
83:- `:master` : which will pull an image in sync with git's [master](https://github.com/pomerium/pomerium/tree/master) branch.
86:  docker pull pomerium/pomerium:master
109:See the [README](https://github.com/pomerium/pomerium-helm/blob/master/charts/pomerium/README.md) for up to date install options.

integration/clusters/google-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/clusters/google-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/gitlab-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/oidc-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/auth0-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/clusters/ping-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/tpl/backends/pomerium.libsonnet
138:  local image = 'pomerium/pomerium:${POMERIUM_TAG:-master}',

integration/clusters/oidc-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/oidc-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/clusters/auth0-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/github-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/google-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/google-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/tpl/backends/k3s.libsonnet
10:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/tpl/backends/mock-idp.libsonnet
5:  local image = 'pomerium/mock-idps:${MOCK_IDPS_TAG:-master}',

integration/clusters/gitlab-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/clusters/github-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/clusters/onelogin-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/github-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/azure-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/onelogin-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/clusters/onelogin-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/okta-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/okta-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/azure-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/clusters/gitlab-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/ping-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/auth0-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/ping-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

integration/clusters/okta-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/auth0-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/ping-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/onelogin-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/auth0-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/ping-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/github-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/oidc-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/oidc-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/gitlab-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/google-multi/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
181:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
240:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
298:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
356:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/azure-nginx/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
400:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/gitlab-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/okta-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/onelogin-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/okta-kubernetes/compose.yml
11:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then
84:                  "image": "pomerium/mock-idps:${MOCK_IDPS_TAG:-master}",
203:                  "image": "pomerium/pomerium:${POMERIUM_TAG:-master}",
1015:      if [ "${POMERIUM_TAG:-master}" = "dev" ]; then

RELEASING.md
10:6. Create a new release branch for major/minor releases, eg `0-15-0`, off master.  Patch releases use existing release branches.
27:GITHUB_TOKEN=XXXXXX ./changelog.sh v0.14.0 v0.15.0 master changelog.out
33:GITHUB_TOKEN=XXXXXX ./changelog.sh v0.14.6 v0.14.7 master changelog.out

integration/clusters/azure-traefik/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
176:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/azure-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}

integration/clusters/github-single/compose.yml
113:    image: pomerium/mock-idps:${MOCK_IDPS_TAG:-master}
175:    image: pomerium/pomerium:${POMERIUM_TAG:-master}
```

**Comment by travisgroth** [comment]
> I believe there are still many several references in the integration tests (though the duplicate tag should avoid breakage I think?) and docs. Maybe best to do in one sweep?

Ah, I missed the `.yml` files entirely.  There's a lot of noise but yes I think I can catch more of these.  I'll take a second pass.

**Review by desimone — APPROVED** [review]
LGTM, but I might have missed some doc references.

File: docs/reference/readme.md [hunk 1]
```diff
@@ -524,53 +524,53 @@ Expose a prometheus endpoint on the specified port.
 
 Each metric exposed by Pomerium has a `pomerium` prefix, which is omitted in the table below for brevity.
 
-Name                                          | Type      | Description
```
[the remark this answers is not in the document]
Comment (travisgroth) [reply docs/reference/readme.md]: No, that was my editor changing things off screen.  Regenerated.

**Review by alexfornuto — APPROVED** [review]

**Pull request merged**